query-service-impl/src/main/java/org/hypertrace/core/query/service/pinot/QueryRequestToPinotSQLConverter.java

```diff
@@ -269,7 +270,7 @@ private String convertExpressionToString(
             viewDefinition.getPhysicalColumnNames(getLogicalColumnName(expression));
         return joiner.join(columnNames);
       case ATTRIBUTE_EXPRESSION:
-        if (isMapAttributeExpression(expression)) {
+        if (isAttributeExpressionMapAttribute(expression)) {
```

Comment:
Now this condition is correct, just incorrectly named (It's a map selection, not any map attribute).